packages/api/test/pin.spec.js

```diff
@@ -568,27 +723,30 @@ describe('Pinning APIs endpoints', () => {
       assert.strictEqual(getEffectivePinStatus(pins), 'queued')
     })
 
-    it('should return "queued" at least 1 pin has remote status', () => {
+    it('returns "queued" at least 1 pin has remote status', () => {
       const pins = [
         createPinWithStatus('UnpinQueued'),
         createPinWithStatus('PinError'),
-        createPinWithStatus('PinQueued')
+        createPinWithStatus('Remote')
       ]
-
       assert.strictEqual(getEffectivePinStatus(pins), 'queued')
     })
 
-    it('should return failed pins have statuses other than Pinned, Pinning, PinQueued or Remote', () => {
+    it('returns "failed" if pins have statuses other than Pinned, Pinning, PinQueued or Remote', () => {
       const pins = [
         createPinWithStatus('UnpinQueued'),
         createPinWithStatus('PinError')
       ]
-
       assert.strictEqual(getEffectivePinStatus(pins), 'failed')
     })
   })
 
-  describe('DELETE /pins/:requestId', () => {
+  describe.skip('DELETE /pins/:requestId', () => {
```

Comment:
This shouldn't be skipped, isn't it?
Also this is somehow duplicated, can you please double check (see https://github.com/web3-storage/web3.storage/pull/705/files#diff-14c9b706d6ea2d7b7b4b2742105bb323beb6b3fdb23b862926505fcf482eb37aR651)

Reply:
Have reinstated this and removed duplicates.